react/wbooks/src/app/screens/SignUp/reducer/reducer.js

```diff
@@ -0,0 +1,10 @@
+export const initialState = { user: {} };
+
+export default function reducer(state, action) {
+  switch (action.type) {
+    case 'createUser':
+      return { user: action.payload };
+    default:
```

Comment:
you can use state.merge and pass to this the part of the state that is actually changing.
`state.merge({user:action.payload});`

Reply:
i tried to put state.merge but the browser said state.merge is not a function.

![images error state merge](https://user-images.githubusercontent.com/42941989/67062529-e8a3f580-f139-11e9-9065-cf7ece74dfba.png)

i think that state.merge works in redux, but it isn't work in useReducer. I looking for about it but i didn't find anything.